Applications/MIDIMonitor/SMMCombinationInputStream.m

```diff
@@ -139,7 +139,7 @@ - (NSSet *)selectedInputSources
 
     return inputSources;
 }
-
+//WORK HERE
```

Comment:
Please remove things like this before putting up a PR.

Reply:
Sorry I missed this. I've never made a real PR until now.

Reply:
No worries! You have to start somewhere.